container/apptabs.go

```diff
@@ -35,6 +39,7 @@ func NewAppTabs(items ...*TabItem) *AppTabs {
 			BaseWidget: widget.BaseWidget{},
 			current:    -1,
 		},
+		Items: items,
```

Comment:
I don't agree with this duplication of data if it can be avoided.
If it cannot be avoided then you are missing setters in `Refresh` where the data could have changed due to external updates.

Reply:
Fixed